plugins/modules/netbox_virtual_machine.py

```diff
@@ -118,6 +118,12 @@
           - configuration context of the virtual machine
         required: false
         type: dict
+      comments:
+        description:
+          - Comments of the virtual machine
+        required: false
+        type: str
+      custom_fields:
```

Comment:
Do you mind removing L126?

Reply:
Oh, sure, this field is appeared by wrong copy-paste
Fixed